atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -35,6 +37,7 @@ class FetchSourcesPlugin(PreBuildPlugin):
     is_allowed_to_fail = False
     SRPMS_DOWNLOAD_DIR = 'image_sources'
     REMOTE_SOUCES_DOWNLOAD_DIR = 'remote_sources'
+    MAVEN_SOUCES_DOWNLOAD_DIR = 'maven_sources'
```

Comment:
typo  SOUCES vs SOURCES

we have typo also in REMOTE_SOUCES_DOWNLOAD_DIR

Reply:
fixed the typos.